pkg/util/kubernetes.go

```diff
@@ -571,3 +576,42 @@ func NewCheckJob(restic *api.Restic, hostName, smartPrefix string, image docker.
 
 	return job
 }
+
+func GetOriginalReplicaFromAnnotation(kind string, obj interface{}) (int32, error) {
```

Comment:
Do we need full object here? Only annotation is enough.

Reply:
ok GetIntValue() is used